### PR TITLE
Add Svelte version of events multiple template

### PR DESCRIPTION
**Pull request by emma-imber**
## What does this change?
Adds a Svelte version of the events multiple template. Also renames/creates some reusable Svelte components to make future Svelte migrations easier to do.

Also renames the template GAM variables to be more descriptive and user friendly.

### Mobile
Screenshot 2023-11-06 at 12 35 42

### Tablet
Screenshot 2023-11-06 at 12 35 54

### Desktop
Screenshot 2023-11-06 at 12 36 07

### CODE REVIEW

File: src/templates/components/EventsHeader.svelte [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+	}
+
+	div.banner-description-container {
+		margin-top: 20px;
+		margin-bottom: 20px;
+	}
+
+	.button-container {
+		display: flex;
+		display: -webkit-flex;
```
Comment (domlander) [inline src/templates/components/EventsHeader.svelte:78]: Do we still need the vendor prefixes for flexbox?

File: src/templates/components/EventsHeader.svelte [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+		-webkit-box-direction: normal;
```
Comment (domlander) [inline src/templates/components/EventsHeader.svelte:79]: Why are `-webkit-box-direction` and `-webkit-box-orient` required?

Comment (emma-imber) [reply src/templates/components/EventsHeader.svelte]: Yep good point! I was copying the old CSS but it's actually much better to use flex-direction here, so I've updated it to that

File: src/templates/components/EventsHeader.svelte (hunk 1 above)
Comment (emma-imber) [reply src/templates/components/EventsHeader.svelte]: You're right that we don't need it here, will update! We do still need it in some other places I think

File: src/templates/components/ManualCard.svelte [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+		padding: 12px 10px;
+		display: block;
+		margin: 0px;
+		width: 50%;
+	}
+
+	a.card-3,
+	a.card-4 {
+		display: none;
+	}
```
Comment (mxdvl) [inline src/templates/components/ManualCard.svelte:65]: Would this not be better handled by an [`:nth-of-type` selector](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-of-type)?

File: src/templates/components/fonts/EgyptianWeb.css [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+	src: url('https://assets.guim.co.uk/static/frontend/fonts/guardian-headline/noalts-not-hinted/GHGuardianHeadline-Medium.woff2?http3=true')
+			format('woff2'),
+		url('https://assets.guim.co.uk/static/frontend/fonts/guardian-headline/latin1-not-hinted/GHGuardianHeadline-Medium.woff?http3=true')
+			format('woff'),
+		url('https://assets.guim.co.uk/static/frontend/fonts/guardian-headline/latin1-not-hinted/GHGuardianHeadline-Medium.ttf?http3=true')
+			format('truetype');
+	font-weight: 500;
+	font-style: normal;
+	font-display: swap;
+}
```
Comment (mxdvl) [inline src/templates/components/fonts/EgyptianWeb.css:12]: Is this taken from https://github.com/guardian/fonts?

File: src/templates/components/icons/LiveLogo.svelte [hunk 5]
```diff
@@ -0,0 +1,58 @@
+<svg
+	aria-hidden="true"
+	class="icon icon--guardian-live-logo"
+	x="0px"
+	y="0px"
```
Comment (mxdvl) [inline src/templates/components/icons/LiveLogo.svelte:5]: You can remove these attributes along with the `class`

File: src/templates/components/icons/LiveLogo.svelte [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+	width="120.9px"
+	height="63.9px"
+	viewBox="0 0 120.9 63.9"
+	fill="#ffffff"
+	><g>
```
Comment (mxdvl) [inline src/templates/components/icons/LiveLogo.svelte:10]: This `g` group is not really doing anything.

**Review by mxdvl — COMMENTED** [review]
I think you can lean a bit more into CSS to and flex to hide cards below a certain breakpoint.

File: src/templates/components/EventsHeader.svelte [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+	}
+
+	.banner-description {
+		font-size: 16px;
+		line-height: 22px;
+		font-family: 'Guardian Egyptian Web', 'Georgia', serif;
+		color: #000000;
+		background: #ffffff;
+		box-decoration-break: clone;
+		-webkit-box-decoration-break: clone;
```
[the remark this answers is not in the document]
Comment (emma-imber) [reply src/templates/components/EventsHeader.svelte:62]: It handles the styling of the description text here:

With box-decoration-break:
Screenshot 2023-11-08 at 09 46 19

Without:
Screenshot 2023-11-08 at 09 46 33

But open to alternative solutions that have better support!

File: src/templates/components/fonts/EgyptianWeb.css (hunk 4 above)
Comment (emma-imber) [reply src/templates/components/fonts/EgyptianWeb.css:12]: Ah good spot - this font was actually in the Headline file but the legacy template had it under a different name. I've now deleted this font face and am using the one from the Headline font file.

File: src/templates/components/ManualCard.svelte (hunk 3 above)
Comment (emma-imber) [reply src/templates/components/ManualCard.svelte]: Yeah nice idea! Implemented 🙂

File: src/templates/components/EventsHeader.svelte (hunk 7 above)
[the remark this answers is not in the document]
Comment (mxdvl) [reply src/templates/components/EventsHeader.svelte:62]: Very nice!

**Review by chrislomaxjones — APPROVED** [review]
This looks great! And a massive improvement over the legacy templates.

It's really nice to be able to check it out with the development server as well 🎉

Screenshot 2023-11-14 at 15 03 08

**Comment by SiAdcock** [comment]
Awesome stuff @emma-imber 💎 

Can we also delete the [legacy template](https://github.com/guardian/commercial-templates/tree/main/legacy/src/events-multiple) as part of this?

**Comment by emma-imber** [comment]
> Awesome stuff @emma-imber 💎
> 
> Can we also delete the [legacy template](https://github.com/guardian/commercial-templates/tree/main/legacy/src/events-multiple) as part of this?

I might split this out into a separate PR just because this one is already pretty chunky!

**Pull request merged**